qdev_wrappers/show_num.py

```diff
@@ -4,46 +4,110 @@
 from qcodes.plots.qcmatplotlib import MatPlot
 
 
-def show_num(id, useQT=False, do_plots=True, **kwargs):
+def show_num(id, useQT=False, ave_col=False, ave_row=False, do_plot=True, savepng=False):
```

Comment:
Did you change `do_plots` to `do_plot` on purose or is this a typo? We might consider keeping the old version to not break compatibility for everybody else.

Reply:
I think the version of show_num I had was slightly different than the one in qdev-wrappers. I have no preference for naming so feel free to change.